common/defines.h

```diff
@@ -33,3 +33,9 @@
     } while(0)
 
 #define TDNF_DEFAULT_MAX_STRING_LEN       16384000
+
+/*
+ * creating this under /var/run because /var/run/lock doesn't exist
+ * in fedora docker images and as a result ci fails
+ */
+#define TDNF_INSTANCE_LOCK_FILE     "/var/run/.tdnf-instance-lockfile"
```

Comment:
Don't declare this macro here again. There should be only one declaration.

It's already there in `client/defines.h`

Reply:
I think code from `common/` does not include fles from`client/`. So the macro should be moved from `client/defines.h` to `common/defines.h`, if it's needed in `common/`

Reply:
Good call, moved the defines to common.